browser-test/src/question_lifecycle.test.ts

```diff
@@ -4,21 +4,19 @@ import {
   AdminQuestions,
   AdminPrograms,
   endSession,
-  dropTables,
   seedCanonicalQuestions,
   waitForPageJsLoad,
+  validateScreenshot,
 } from './support'
 import {QuestionType} from './support/admin_questions'
+import {BASE_URL} from './support/config'
 
 describe('normal question lifecycle', () => {
-  beforeAll(async () => {
-    const {page} = await startSession()
-    await dropTables(page)
+  it('canonical question seeding works', async () => {
+    const {browser, page} = await startSession(/* clearDb= */ true)
```

Comment:
Why move this out of beforeAll?

Reply:
There is no need in beforeAll now. This test drop tables on each startSession invocation and `seedCanonicalQuestions` is needed only by one test. And even there I think it's not really useful but sure, why not have test that verifies that seeding works.

Reply:
Oh sorry, I meant why not keep the startSession() call (and maybe some other shared setup) in a beforeEach?

Reply:
Ah, ok. Looks like this test used different style - starting session on each test case, while other tests start session once and reset it between test cases. I refactored this test to follow same style. And moved 'clearDb' logic to the resetSession method.